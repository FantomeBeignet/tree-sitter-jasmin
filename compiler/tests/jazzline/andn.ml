open Common

let () = parse_and_print "andn.jazz"  "andn"
