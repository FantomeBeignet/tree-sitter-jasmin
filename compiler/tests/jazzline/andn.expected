proc andn(h_138@uint64, b_132@uint64, c_133@uint64) = 
{
 true && true
}
not TMP_____149@uint64 b_132@uint64;
and h_138@uint64 c_133@uint64 TMP_____149@uint64;
{
 true && true
}
 